cloud-yarn/src/main/java/com/sequenceiq/cloudbreak/cloud/yarn/YarnMetadataCollector.java

```diff
@@ -138,7 +143,21 @@ private CloudResource getYarnApplcationResource(Iterable<CloudResource> resource
 
     @Override
     public List<CloudLoadBalancerMetadata> collectLoadBalancer(AuthenticatedContext ac, List<LoadBalancerType> loadBalancerTypes) {
-        // no-op
-        return Collections.emptyList();
+        YarnClient yarnClient = yarnClientUtil.createYarnClient(ac);
+        String loadBalancerApplicationName = applicationNameUtil.createLoadBalancerName(ac);
+        Iterable<Container> loadBalancerContainers = getContainers(loadBalancerApplicationName, yarnClient);
+        Iterator<Container> containerIterator = loadBalancerContainers.iterator();
+
+        List<CloudLoadBalancerMetadata> loadBalancerMetadata = Lists.newArrayList();
+        for (LoadBalancerType loadBalancerType : loadBalancerTypes) {
+            Container container = containerIterator.next();
+            CloudLoadBalancerMetadata metadata = new CloudLoadBalancerMetadata.Builder()
+                    .withType(loadBalancerType)
+                    .withName(loadBalancerApplicationName + loadBalancerType.name())
```

Comment:
Where is this naming convention being set? If it's used in other places, then we should have a shared method that can be called to generate this, so that it doesn't get out of sync if we change it one place and forget to update it elsewhere.